airbyte-webapp/src/components/JobItem/components/ResetStreamDetails.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import styled from "styled-components";
+
+interface IProps {
+  names: string[];
+}
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  margin-top: 4px;
+`;
+
+const BadgeContainer = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 20px;
+  padding: 0 5px;
+  background: #f8f8fa;
+  border: 1px solid #e8e8ed;
+  border-radius: 4px;
+`;
+
+const BadgeText = styled.p`
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 15px;
+  color: #8b8ba0;
+`;
+
+export const ResetStreamsDetails: React.FC<IProps> = ({ names }) => (
+  <Container>
+    {names.map((name) => (
+      <BadgeContainer>
+        <BadgeText>{name}</BadgeText>
```

Comment:
Our designs suggest that we hide all reset "tags" that are overflowing. This code currently renders them all out. Could we make sure to ellipsis in the end, when there are too many. https://developpaper.com/tips-research-on-css-whole-block-text-overflow-and-ellipsis/ has some good examples how to do that with tags like we have.